### DISCUSSION TITLE
Can I use EJS Instead of Handlebars?

### DISCUSSION THREAD

**Discussion by techn0mate** [lifeofsoumya/StockAlert#2] [Ideas]
I am familiar with EJS syntax, is it possible to use that as an alternative of HBS used in this project?

**Comment by lifeofsoumya** [comment]
Of course, you can use EJS. Though EJS is a popular library compared to HBS, I've used it for two major reasons.

**1. Simplicity is using variables:** I haven't got to use any JS syntax inside the file which could've been easier with EJS, for that reason I found that using variables using only {{}} is much easier than <%= %>.
**2. Nodemailer HBS:** To render the dynamic mail, previously I've used a package that uses NodeMailer-HBS integration. For that reason, I continued using HBS to render the index.
